packages/react/cypress/support/commands.js

```diff
@@ -0,0 +1,3 @@
+const compareSnapshotCommand = require('cypress-image-diff-js/dist/command');
+
```

Comment:
Shouldn't `import '@testing-library/cypress/add-commands';` be in here? 

Perhaps it doesn't matter since we import this file to cypress/support/index.js anyway...

https://testing-library.com/docs/cypress-testing-library/intro/

Reply:
Definitely. moved the import here